src/main/java/pl/javastart/task/Drink.java

```diff
@@ -0,0 +1,15 @@
+package pl.javastart.task;
+
+public class Drink {
+    String drinkName;
+    double drinkPrice;
+    boolean drinkAlcohol;
```

Comment:
Dkoro jesteśmy w klasie Drink, to nie ma już potrzeby dodawania "drink" do nazwy każdego pola